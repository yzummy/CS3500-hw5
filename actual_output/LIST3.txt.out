
___Entering new scope...

TOKEN: LIST 		 LEXEME: list
TOKEN: LPAREN 		 LEXEME: (
TOKEN: INTCONST 		 LEXEME: 23
CONST -> INTCONST
TOKEN: COMMA 		 LEXEME: ,
TOKEN: FLOATCONST 		 LEXEME: 3.14
CONST -> FLOATCONST
TOKEN: COMMA 		 LEXEME: ,
TOKEN: TRUE 		 LEXEME: TRUE
CONST -> TRUE
TOKEN: COMMA 		 LEXEME: ,
TOKEN: STRCONST 		 LEXEME: "BoJack"
CONST -> STRCONST
TOKEN: COMMA 		 LEXEME: ,
TOKEN: FLOATCONST 		 LEXEME: -98.689
CONST -> FLOATCONST
TOKEN: COMMA 		 LEXEME: ,
TOKEN: FALSE 		 LEXEME: FALSE
CONST -> FALSE
TOKEN: COMMA 		 LEXEME: ,
TOKEN: STRCONST 		 LEXEME: "Mr. PeanutButter"
CONST -> STRCONST
TOKEN: RPAREN 		 LEXEME: )
CONST_LIST -> CONST
CONST_LIST -> CONST, CONST_LIST
CONST_LIST -> CONST, CONST_LIST
CONST_LIST -> CONST, CONST_LIST
CONST_LIST -> CONST, CONST_LIST
CONST_LIST -> CONST, CONST_LIST
CONST_LIST -> CONST, CONST_LIST
LIST_EXPR -> LIST ( CONST_LIST )
EXPR -> LIST_EXPR
START -> EXPR

---- Completed parsing ----

Value of the expression is: ( 23 3.14 TRUE "BoJack" -98.69 FALSE "Mr. PeanutButter" )

___Exiting scope...

