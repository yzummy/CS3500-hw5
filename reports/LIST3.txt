
---- Completed parsing ----					---- Completed parsing ----

Value of the expression is: ( 23 3.14 TRUE "BoJack" -98.69 FA	Value of the expression is: ( 23   3.14 TRUE "BoJack" -98.69 
