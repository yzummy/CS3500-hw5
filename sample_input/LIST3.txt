list(23, 3.14, TRUE, "BoJack", -98.689, FALSE, 
     "Mr. PeanutButter")
