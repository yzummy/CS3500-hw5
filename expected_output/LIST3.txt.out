
---- Completed parsing ----

Value of the expression is: ( 23   3.14 TRUE "BoJack" -98.69 FALSE "Mr. PeanutButter" )
